torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -154,6 +154,8 @@ def generate_ttir(kernel, kwargs):
 
     src = ASTSource(kernel, signature, constants, specialization)
     ttir_module = src.make_ir(options, context)
+    if not ttir_module.verify():
+        raise Exception("Verification for TTIR module has failed")
```

Comment:
What is this? We can get malformed TTIR from Triton?

Reply:
If the user code is malformed, then triton produced malformed TTIR which gets checked just like this at a later time